kubernetes-examples/src/main/java/io/fabric8/openshift/examples/BuildConfigExamples.java

```diff
@@ -37,106 +37,244 @@
+          .withNewMetadata()
+          .withName("custom-build-config")
+          .endMetadata()
+          .withNewSpec()
+          .withServiceAccount("fabric8")
+          .withNewSource()
+          .withType("Git")
+          .withNewGit()
+          .withUri("https://github.com/fabric8io/example-camel-cdi.git")
+          .endGit()
+          .endSource()
+          .withNewResources()
+            .addToLimits("mykey", new Quantity("10"))
+            .addToRequests("mykey", new Quantity("10"))
+          .endResources()
+          .withNewStrategy()
+          .withType("Source")
+          .withNewSourceStrategy()
+          .withNewFrom().withName("java-sti:latest").withKind("DockerImage").endFrom()
+          .endSourceStrategy()
+          .endStrategy()
+          .withNewOutput()
+          .withNewTo().withKind("DockerImage").withName("example-camel-cdi:latest").endTo()
+          .endOutput()
+          .addNewTrigger()
+          .withType("GitHub")
+          .withNewGithub()
           .withSecret("secret101")
+          .endGithub()
+          .endTrigger()
+          .endSpec()
+          .done());
+
+        Build build = client.buildConfigs().inNamespace("thisisatest").withName("custom-build-config").instantiate(new BuildRequestBuilder()
+          .withNewMetadata().withName("custom-build-config").endMetadata()
           .build());
+        log("Build:", build.getMetadata().getName());
+
+        client.buildConfigs().inNamespace("thisisatest").withName("custom-build-config")
+          .withSecret("secret101")
+          .withType("github")
+          .trigger(new WebHookTriggerBuilder()
+            .withSecret("secret101")
+            .build());
+
+
+        Thread.sleep(6000);
+
+        log("Builds:");
+        for (Build b: client.builds().inNamespace("thisisatest").list().getItems()) {
+          log("\t\t\t"+b.getMetadata().getName());
 
+          log("\t\t\t\t\t Log:");client.builds().inNamespace("thisisatest").withName(b.getMetadata().getName()).watchLog(System.out);
+        }
 
-      Thread.sleep(6000);
 
-      log("Builds:");
-      for (Build b: client.builds().inNamespace("thisisatest").list().getItems()) {
-        log("\t\t\t"+b.getMetadata().getName());
 
-        log("\t\t\t\t\t Log:");client.builds().inNamespace("thisisatest").withName(b.getMetadata().getName()).watchLog(System.out);
+        log("Done.");
+      }finally {
+        client.namespaces().withName("thisisatest").delete();
+        client.close();
       }
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *         http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Maybe you copied this accidentally???

Reply:
@rohanKanojia These comment lines were already there in the file. I just added a try block. What is expected now? Should I remove the commented lines?

Reply:
Are you sure about that? I don't see it in master revision:

https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-examples/src/main/java/io/fabric8/openshift/examples/BuildConfigExamples.java

Reply:
Sorry, I was looking at initial lines, didn't noticed repetition in later lines before commenting. I have removed those lines now.